rules/javaee/XmlWebServicesRules.windup.xml

```diff
@@ -61,28 +61,28 @@
         </rule>
         <rule id="XmlWebserviceRules_Cbxnb">
             <when>
-                <xmlfile as="1" matches="/jee:webservices | /j2e:webservices">
+                <xmlfile as="webservices" matches="/jee:webservices | /j2e:webservices">
                     <namespace prefix="j2e" uri="http://java.sun.com/xml/ns/j2ee"/>
                     <namespace prefix="jee" uri="http://java.sun.com/xml/ns/javaee"/>
                 </xmlfile>
-                <xmlfile as="2" matches="//jee:handler-class | //j2e:handler-class" from="1">
+                <xmlfile as="handler-class" matches="//jee:handler-class | //j2e:handler-class" from="webservices">
```

Comment:
Hey Sande,

Since this could also be plural, I suggest using plural "handler-classes" instead.

Thanks for this update!